crates/diesel_models/src/payment_method.rs

```diff
@@ -32,6 +32,7 @@ pub struct PaymentMethod {
     pub payment_method_issuer: Option<String>,
     pub payment_method_issuer_code: Option<storage_enums::PaymentMethodIssuerCode>,
     pub metadata: Option<pii::SecretSerdeValue>,
+    pub card_details: Option<Encryption>,
```

Comment:
Can we give this field a generic name to incorporate both card and bank details?

Reply:
Currently only card details are being considered in this PR as required by Customers PM list API.